tools/telemetry-generator/src/handlers/appInsightsCustomBenchmarkHandler.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TelemetryClient } from "applicationinsights";
+
+/**
+ * This handler emits metrics to the Azure App Insights instance configured by the telemetryClient provided to this handler.
+ * This handler expects the 'telemetryClient' arg to be TelemetryClient class from the 'applicationinsights' Azure package.
+ */
+module.exports = function handler(fileData, telemetryClient: TelemetryClient): void {
+	console.log(`Found ${fileData.benchmarks.length} total benchmark tests to emit`);
+	for (const testData of fileData.benchmarks) {
+		for (const customDataKey of Object.keys(testData.customData)) {
```

Comment:
@Josmithr @alexvy86 

Is there a better practice when iterating through JSON objects?

`testData.customData` : `any`
`trackMetric.value`: `number`

I initially tried using `for (const [customDataKey, customDataValue] of Object.entries(testData.customData))` (`customDataValue: unknown`), and below happens.

<img width="896" alt="Screenshot 2024-08-16 at 16 01 40" src="https://github.com/user-attachments/assets/5196e757-89b0-4e26-aea0-6b504433c4f8">

Using `Object.keys()` bypasses the problem. Wanted to be extra sure when iterating through an object

Reply:
JSON objects aren't typed, unless you run them through some sort of schema validator. So, unfortunately, you're stuck with casting 😕